Shared/Flows/Launch/SplashViewController.swift

```diff
@@ -91,7 +91,7 @@ class SplashViewController: FullScreenViewController, TransitionableViewControll
         self.label.setText(emotion.definition)
         self.label.textColor = emotion.color
         
-        self.emotionNameLabel.setText(emotion.rawValue.capitalized)
+        self.emotionNameLabel.setText(emotion.description.capitalized)
```

Comment:
Emotions now have a description field. This is distinct from their underlying "id" value, which has the spaces and hyphens removed.